addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -23,6 +23,7 @@ def default_get(self, fields):
             if move_ids_use_document:
                 raise UserError(_('You can only reverse documents with legal invoicing documents from Latin America one at a time.\nProblematic documents: %s') % ", ".join(move_ids_use_document.mapped('name')))
 
+        res['move_id'] = move_ids[0].id if move_ids else False
```

Comment:
should already be done by the super?

Reply:
no, it's not there on super. This field lost the functionality on [this commit](https://github.com/odoo/odoo/commit/fe47011c3fdcec0bda5334f10679e853fe41e45a#diff-8274327aa3ab89fe8418252e9ec2b3eaL24)

Reply:
great @jco-odoo 
> @jjscarafia I suggest to do similar in
> 
> https://github.com/odoo/odoo/blob/77c28421e6c7711e3d3c39dbece4e2bfa0b26b5c/addons/account/wizard/account_move_reversal.py#L54
> 
> as what I did in the account_debit_note branch: create a _prepare_default_reversal(move) That way you can just inherit that method.

So in this PR we should split [this](https://github.com/odoo/odoo/blob/13.0/addons/account/wizard/account_move_reversal.py#L54-L61) on a new methoo called "_prepare_default_reversal"?

Reply:
If you can do two separate commits (one for account and one for the l10n_...part?)

Reply:
Remove ```
res['residual'] = len(move_ids) == 1 and move_ids.amount_residual or 0
    res['currency_id'] = len(move_ids.currency_id) == 1 and move_ids.currency_id.id or False
    res['move_type'] = len(move_ids) == 1 and move_ids.type or False
```
from generic and set res['move_id'] there, but only when move_id is not already in res

Reply:
That way the values should be computed.